model/project_parser.go

```diff
@@ -517,6 +520,24 @@ func LoadProjectInto(data []byte, identifier string, project *Project) (*ParserP
 	return intermediateProject, errors.Wrap(err, LoadProjectError)
 }
 
+func GetProjectFromFile(ctx context.Context, ref ProjectRef, file, token string) (*Project, *ParserProject, error) {
```

Comment:
Is this mostly the same as the GetRemoteConfig function?

Reply:
Yes 😮  I'll refactor